### ISSUE TITLE
Feature request: startup configurability

### ISSUE THREAD

**Issue by teaguecl** [wernight/docker-qbittorrent#5]
It would be nice if entrypoint.sh could be extended by the user.  For example, it could execute /config/entrypoint-pre.sh before doing any work (if that file exists), and /config/entrypoint-post.sh after.  My motivation for the request is that I had to modify entrypoint.sh to use a umask of 000 instead of 002.  I could have put it in entrpoint-post.sh and not had to modify the container.

**Comment by wernight** [comment]
It's not standard to have those, as you said people usually then build their own container based on one.

The umask is probably a fair point as I think it's common to wish for qBT to have other masks. Could you just give you motivation for having another mask?

**Comment by teaguecl** [comment]
Fair point regarding the extra scripts.  My desire to change the umask is that I have a process that runs outside the container on the host, which needs to copy, move and/or rename the files that get downloaded.  I could create a user on the host with UID 520, but I really don't need another user account to get what I need done.

**Comment by wernight** [comment]
I made it so that qBT can work with any UID/GID so you can also start with `--user $UID:$GID` (any int:int). Also as long as your external process is running as the same GID (or same group) it also should work.
